### PR TITLE
Convert into md

### PR DESCRIPTION
**Pull request by UESTCzhouyuchuan**
Convert into markdown and sync english version content.
Page:
Web/HTTP/status

### CODE REVIEW

**Comment by yin1999** [comment]
To keep the commit history, please follow the commit step below:

1. commit 1: renamve the `.html` file to `.md` file (using git command `git mv /path/to/index.html /path/to/index.md`)
2. commit 2: convert the content to markdown (and sync with english version, it's optional)

You can refer [the merged PR before](https://github.com/mdn/translated-content/pull/4744/commits)

If you have difficulty to deal with it in this pr, new pr is welcomed.

File: files/zh-cn/web/http/status/index.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+{{HTTPSidebar}}
+
+HTTP 响应状态码用来表明特定 [HTTP](/en-US/docs/Web/HTTP) 请求是否成功完成。
+响应被归为以下五大类:
+
+1. [信息响应](#information_responses) (`100`–`199`)
+2. [成功响应](#successful_responses) (`200`–`299`)
+3. [重定向消息](#redirection_messages) (`300`–`399`)
+4. [客户端错误响应](#client_error_responses) (`400`–`499`)
+5. [服务端错误响应](#server_error_responses) (`500`–`599`)
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:21]: those anchors are not correct.

```md
1. [信息响应](#信息响应) (`100`–`199`)
2. [成功响应](#成功响应) (`200`–`299`)
3. [重定向消息](#重定向消息) (`300`–`399`)
4. [客户端错误响应](#客户端错误响应) (`400`–`499`)
5. [服务端错误响应](#服务端错误响应) (`500`–`599`)
```

In markdown, the head is almost the same as the `id`.

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/http/status/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/http/status/index.md:21]: so, please redo the commits and also update those anchors.

File: files/zh-cn/web/http/status/index.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+- {{HTTPStatus(510, "510 Not Extended")}}
+  - : 服务器需要对请求进行进一步扩展才能完成请求。
+- {{HTTPStatus(511, "511 Network Authentication Required")}}
+  - : 指示客户端需要进行身份验证才能获得网络访问权限。
+
+## 浏览器兼容性
+
+{{Compat("http.status")}}
+
+## 你也可以看看
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:180]: suggestion:

```markdown
## 参见
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by UESTCzhouyuchuan** [comment]
> To keep the commit history, please follow the commit step below:
> 
> 1. commit 1: renamve the `.html` file to `.md` file (using git command `git mv /path/to/index.html /path/to/index.md`)
> 2. commit 2: convert the content to markdown (and sync with english version, it's optional)
> 
> You can refer [the merged PR before](https://github.com/mdn/translated-content/pull/4744/commits)
> 
> If you have difficulty to deal with it in this pr, new pr is welcomed.
Thanks. My friend. I have taken your advice.

File: files/zh-cn/web/http/status/index.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+1. [信息响应](#信息响应) (`100`–`199`)
+2. [成功响应](#成功响应) (`200`–`299`)
+3. [重定向消息](#重定向消息) (`300`–`399`)
+4. [客户端错误响应](#客户端错误响应) (`400`–`499`)
+5. [服务端错误响应](#服务端错误响应) (`500`–`599`)
+
+以下状态码由 [section 10 of RFC 2616](https://datatracker.ietf.org/doc/html/rfc2616#section-10)定义。你可以在[RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-6)中找到更新后的规范。
+
+> **Note:** 如果您收到的响应不在此列表中 [this list](#information-responses)，则它为非标准响应，可能是服务器软件的自定义响应。
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:22]: For hightlight this notecard, using `> **备注：** ` (in `zh-CN`) as prefix. Reference: [markdown in mdn](https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN#%E5%A4%87%E6%B3%A8%E3%80%81%E8%AD%A6%E5%91%8A%E5%92%8C%E6%A0%87%E6%B3%A8)

And this anchor is not right now.

suggestion:

```markdown
> **备注：** 如果您收到的响应不在[此列表](#信息响应)中，...
```

File: files/zh-cn/web/http/status/index.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+- {{HTTPStatus(511, "511 Network Authentication Required")}}
+  - : 指示客户端需要进行身份验证才能获得网络访问权限。
+
+## 浏览器兼容性
+
+{{Compat("http.status")}}
+
+## 参见
+
+- [List of HTTP status codes on Wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:179]: `zh-CN` wiki is prefered (if exists):

```markdown
- [维基百科上的 HTTP 状态码](https://zh.wikipedia.org/wiki/HTTP状态码)
```

File: files/zh-cn/web/http/status/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+title: HTTP 响应代码
+slug: Web/HTTP/Status
+tags:
+  - HTTP
+  - 状态码
+translation_of: Web/HTTP/Status
+---
+{{HTTPSidebar}}
+
+HTTP 响应状态码用来表明特定 [HTTP](/en-US/docs/Web/HTTP) 请求是否成功完成。
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:11]: just using the link to `zh-CN` (even if the document does not exist, mdn will fallback to `en-US` for not existing document).

```markdown
[HTTP](/zh-CN/docs/Web/HTTP)
```

File: files/zh-cn/web/http/status/index.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+## 信息响应
+
+- {{HTTPStatus(100, "100 Continue")}}
+  - : 这个临时响应表明，迄今为止的所有内容都是可行的，客户端应该继续请求，如果已经完成，则忽略它。
+- {{HTTPStatus(101, "101 Switching Protocols")}}
+  - : 该代码是响应客户端的 {{HTTPHeader("Upgrade")}} 请求头发送的， 指明服务器即将切换的协议。
+- {{HTTPStatus(102, "102 Processing")}} ({{Glossary("WebDAV")}})
+  - : 此代码表示服务器已收到并正在处理该请求，但当前没有响应可用。
+- {{HTTPStatus(103, "103 Early Hints")}}
+  - : 此状态代码主要用于与 {{HTTPHeader("Link")}} 链接头一起使用，以允许用户代理在服务器准备响应阶段时开始预加载[preloading](/en-US/docs/Web/HTML/Link_types/preload)资源。
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:33]: also, use hyperlink to `zh-CN`:

```markdown
[预加载（preloading）](/zh-CN/docs/Web/HTML/Link_types/preload)
```

File: files/zh-cn/web/http/status/index.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+- {{HTTPStatus(402, "402 Payment Required")}} {{experimental_inline}}
+  - : 此响应代码保留供将来使用。创建此代码的最初目的是将其用于数字支付系统，但是此状态代码很少使用，并且不存在标准约定。
+- {{HTTPStatus(403, "403 Forbidden")}}
+  - : 客户端没有访问内容的权限；也就是说，它是未经授权的，因此服务器拒绝提供请求的资源。与`401 Unauthorized`不同，服务器知道客户端的身份。
+- {{HTTPStatus(404, "404 Not Found")}}
+  - : 服务器找不到请求的资源。在浏览器中，这意味着无法识别URL。在API中，这也可能意味着端点有效，但资源本身不存在。服务器也可以发送此响应，而不是`403 Forbidden`，以向未经授权的客户端隐藏资源的存在。这个响应代码可能是最广为人知的，因为它经常出现在网络上。
+- {{HTTPStatus(405, "405 Method Not Allowed")}}
+  - : 服务器知道请求方法，但目标资源不支持该方法。例如，API可能不允许调用`DELETE`来删除资源。
+- {{HTTPStatus(406, "406 Not Acceptable")}}
+  - : 当web服务器在执行服务器驱动的内容协商[server-driven content negotiation](/en-US/docs/Web/HTTP/Content_negotiation#server-driven_negotiation)后，没有发现任何符合用户代理给定标准的内容时，就会发送此响应。
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:101]: suggestion:

```markdown
[服务端驱动的内容协商](/zh-CN/docs/Web/HTTP/Content_negotiation#服务端驱动型内容协商机制)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/http/status/index.md (hunk 7 above)
Comment (UESTCzhouyuchuan) [reply files/zh-cn/web/http/status/index.md:101]: ok，get it

**Comment by yin1999** [comment]
If there is not conflict, [merging branch](https://github.com/mdn/translated-content/pull/4807/commits/aef70d12836795774f70136e542a70ffeb399d70) is not necessary. Just merge the last three commits into a single commit (without merging `main` branch).

File: files/zh-cn/web/http/status/index.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+- {{HTTPStatus(511, "511 Network Authentication Required")}}
+  - : 指示客户端需要进行身份验证才能获得网络访问权限。
+
+## 浏览器兼容性
+
+{{Compat("http.status")}}
+
+## 参见
+
+- [List of HTTP status codes on Wikipedia](https://zh.wikipedia.org/wiki/List_of_HTTP_status_codes)
```
Comment (yin1999) [inline files/zh-cn/web/http/status/index.md:183]: 这个链接不正确哈，https://github.com/mdn/translated-content/pull/4807#discussion_r834874138

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/http/status/index.md (hunk 8 above)
Comment (UESTCzhouyuchuan) [reply files/zh-cn/web/http/status/index.md:183]: en version content
```md
- [List of HTTP status codes on Wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)
- [IANA official registry of HTTP status codes](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml)
```

我这边用不了wikipedia，算了这搞的，等我研究复试结束后再想办法吧

Comment (UESTCzhouyuchuan) [reply files/zh-cn/web/http/status/index.md]: 链接ok了，哥

Comment (yin1999) [reply files/zh-cn/web/http/status/index.md]: 链接内容请一并帮助修改：

```markdown
- [维基百科上的 HTTP 状态码](https://zh.wikipedia.org/wiki/HTTP状态码)
```

**Pull request merged**